Assets/Scripts/Controllers/WorldController.cs

```diff
@@ -32,8 +32,8 @@ public class WorldController : MonoBehaviour
     public KeyboardManager keyboardManager;
     public CameraController cameraController;
     public SpawnInventoryController spawnInventoryController;
-    public AutosaveManager autosaveManager;
```

Comment:
When i merged with upstream, it complained that the autosave manager didnt exist